content/Tjenester/fiksprotokoll/brukerveiledning/huskeliste_for_opprette_protokoll.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 1. Huskeliste før man starte å sette opp protokoll
+date: 2022-09-23
+alias: [/fiks-plattform/tjenester/fiksprotokoll/brukerveiledning/huskeliste]
+---
+
+## Hvorfor
+
+Før man begynner på arbeidet med å sette opp Fiks Protokoll er det lurt å sjekke at man har alt man trenger og personer med riktig tilgang tilgjengelig.
+
+## Personer
+
+### Person fra kunden
+Det må være en person fra kunden organisasjonen (kommunen) som skal ha opprettet protokoll konto og system som kan logge seg på Fiks forvaltning og gjøre minstekravet for oppsett.
```

Comment:
Litt uklar tekst her.

Reply:
Ser den. Se siste commit @helgeskjellevik

Reply:
Hva med denne endringen i tillegg?

_Det må være en person fra kundeorganisasjonen (kommunen) som skal opprette protokoll system/konto, og som da kan logge seg på Fiks forvaltning og gjøre minstekravet for oppsett._

Reply:
Done